README.rst

```diff
@@ -41,33 +41,31 @@
 .. end-badges
 
 ==================
-Explorepy overview
+``explorepy`` overview
 ==================
 
-Explorepy is an open-source Python API designed to collect and process biosignal data using Mentalab’s Explore device. Amongst other things, Explorepy provides the following features:
-
-* Real-time streaming of ExG, orientation and environmental data
-* Real-time visualization
-* Data recording in CSV and BDF+ formats
-* LSL integration
-* Impedance measurement
-* Explore device configuration
+``explorepy`` is an open-source Python API designed to collect and process ExG data using Mentalab's Explore device. Amongst other things, ``explorepy`` provides the following features:
 
+* Real-time streaming of ExG, orientation and environmental data.
+* Real-time visualization of ExG, orientation and environmental data.
+* Data recording in CSV and BDF+ formats.
+* Integration with LabStreaming Layer.
+* Electrode impedance measurements.
+* Explore device configuration.
 
 Quick installation
 ==================
-For Windows users, the best way to install Explorepy is to download the latest Explorepy version from the
-the `release page <https://github.com/Mentalab-hub/explorepy/releases>`_ or use pip. Please note that dependencies will install automatically from the release page.
+For Windows users, the best way to install ``explorepy`` is to download the latest ``explorepy`` version from the `release page <https://github.com/Mentalab-hub/explorepy/releases>`_ or use pip. Please note that dependencies will install automatically from the release page.
```

Comment:
For Windows users, the best way to install ``explorepy`` is to download the setup file of the latest ``explorepy`` version from the `release page <https://github.com/Mentalab-hub/explorepy/releases>`_ . Please note that dependencies will install automatically from the release page.